app/models/auth_sources/auth_source_external.rb

```diff
@@ -1,4 +1,7 @@
 class AuthSourceExternal < AuthSource
+  include Taxonomix
+  scoped_search :on => :name, :complete_value => :true
```

Comment:
seems to be also defined in the AuthSource parent class, do we need to defined it here again?